core/src/main/resources/reference.conf

```diff
@@ -134,6 +134,18 @@ datastax-java-driver {
     # Overridable in a profile: yes
     page-size = 5000
 
+    # Disable skipping metadata for every query
```

Comment:
If i understand correctly this change modifies the skip only for BoundStatements.
Other queries will still skip.

Reply:
That is correct, will fix it.